packages/babel-parser/src/tokenizer/index.js

```diff
@@ -407,7 +407,7 @@ export default class Tokenizer extends ParserErrors {
     }
 
     const nextPos = this.state.pos + 1;
-    const next = this.input.charCodeAt(nextPos);
+    const next = this.input.codePointAt(nextPos);
```

Comment:
Isn't this slower?

Reply:
It won't be significantly slower even if V8 did not inline `codePointAt`. I can come up with our own `codePointAt` like we did in https://github.com/babel/babel/pull/13211#discussion_r621390095.